CHANGES_NEXT_RELEASE

```diff
@@ -1,3 +1,5 @@
+- Fix: explicitAttrs of device was tained even if not defined
+- Fix: do not include static, lazy and commands in complete registrations to avoid duplicate them in device (#1377) 
```

Comment:
Does it requires modifications to documentation? Or maybe documentation doesn't describe the "internals" of the registration process?

@mapedraza what do you think?

Reply:
Maybe doc should be update, since this was reported as bug but there were tests about that previous behaviour

Reply:
> Does it requires modifications to documentation? Or maybe documentation doesn't describe the "internals" of the registration process?
> 
> @mapedraza what do you think?

As described here, this is the expected behaviour, so, IMHO, there is nothing to change into the documentation
https://github.com/telefonicaid/iotagent-node-lib/blob/0ab531d30587d6d7415b770e0f9072fce3f910d0/doc/api.md?plain=1#L149-L151